enclave-runtime/Makefile

```diff
@@ -44,9 +44,9 @@ else
 endif
 
 ifeq ($(SGX_PRODUCTION), 1)
-	ENCLAVE_FEATURES = --features=production,$(WORKER_MODE)
+	ENCLAVE_FEATURES = --features=production,$(WORKER_MODE),$(ADDITIONAL_FEATURES)
 else
-	ENCLAVE_FEATURES = --features=test,$(WORKER_MODE)
+	ENCLAVE_FEATURES = --features=test,$(WORKER_MODE),$(ADDITIONAL_FEATURES)
```

Comment:
Is the information on how to build / start the worker now documented somewhere? If not, can you add this? Mby directly in the README of the worker, or the book, or whatever is now the standard.

Reply:
created new issue: https://github.com/integritee-network/worker/issues/936